src/main/java/core/basesyntax/dao/ReportWorker.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.fileworker.InputOutputReport;
+import core.basesyntax.service.Validation;
+import java.time.LocalDate;
+import java.util.Map;
+
+public class ReportWorker implements WarehouseDao {
```

Comment:
Dao is a layer in the application that is responsible only for operations related to the database. So let's change the name of this interface.